miri

```diff
@@ -174,6 +174,10 @@ test|test-debug|bless|bless-debug)
     # First build and get a sysroot.
     $CARGO build $CARGO_BUILD_FLAGS --manifest-path "$MIRIDIR"/Cargo.toml
     find_sysroot
+    # rebuild the C test shared object file if it's not there
+    if [ ! -f "tests/external_C/libtestlib.so" ]; then
+        gcc -shared -o tests/external_C/libtestlib.so tests/external_C/test.c
```

Comment:
At a minimum this should be $CC, and we should consider using `cc` in a `build.rs` if appropriate.

Reply:
good idea -- i've made a `CC` variable for now